### ISSUE TITLE
the read_alm documentation is quite confusing

### ISSUE THREAD

**Issue by hivon** [healpy/healpy#928]
The read_alm documentation says
```
hdu : int, or tuple of int, optional
  The header to read. Start at 0. Default: hdu=1
```
when actually is the HDU to be read, not the header
and more importantly it starts at 1, not 0.
Ie, reading all 3 HDUs of a polarized alm file is hdu=(1,2,3)

**Comment by mreineck** [comment]
You are completely correct about HDU vs. header.

Concerning "it starts at 1, not 0": I think the documentation means to say "incontrast to the typical FITS convention,where the primary image counts as HDU 1, we start counting at 0, i.e. the primary image counts as HDU 0."

**Comment by hivon** [comment]
possibly, but you can safely assume that most users do not know or care about the FITS convention, and will only encounter alm FITS files in which the data their care about start at hdu=1.
I think giving an example like
"hdu=1 (or hdu=(1)) will return the alm_T, 
hdu=(1,2,3) will return alm_T, alm_E, alm_B when reading a FITS file containing polarized alm"
will avoid much confusion.

**Comment by mreineck** [comment]
Absolutely! This was meant more as an explanation han an excuse :-) The docstring really is too terse.

**Comment by zonca** [comment]
thanks for reporting this, please take a look at https://github.com/healpy/healpy/pull/930

**Comment by zonca** [comment]
@hivon  thanks for reporting this, please take a look at #930

**Comment by hivon** [comment]
@zonca: #930 seems to fix it. Thanks
